### PR TITLE
Project Review

### PR DESCRIPTION
**Pull request by lucy-sees**
Please review my work

### CODE REVIEW

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @lucy-sees ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits
* ✅  Good styling


## Required Changes ♻️

- [ ] Add a .gitignore file.  `why?` because .gitignore file in your repository's root directory to tell Git which files and directories to ignore when you make a commit. To share the ignore rules with other users who clone the repository, commit the .gitignore file in to your repository.

- [ ] Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request.  see this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Created new branch called main_backup
+
+# updated all tracking branches, including main branch
```
[the remark this answers is not in the document]
Comment (lucy-sees) [reply readme.md]: I have made the adjustments.

[the remark this answers is not in the document]
Comment (lucy-sees) [reply readme.md]: Kindly review and let me know if I can proceed.

File: index.html [hunk 2]
```diff
@@ -21,7 +21,7 @@ <h2>Wrote it simpler to easily delete.</h2>
     <nav>
         <ul>
             
-            <li>Hi<i class="fa-sharp fa-solid fa-face-smile-wink"></i></li>
+            <li>Hi <i class="fa-sharp fa-solid fa-face-smile-wink"></i></li>
             <li>Howdy<i class="fa-sharp fa-solid fa-face-cowboy-hat"></i></li>
             <li>Hello</li>
         </ul>
```
Comment (Alfredbis29) [inline index.html:27]: - [ ] Good job so far as required to this project you should just display `Hello Microverse!` but I can see that you put a lot , for this project we just need `h1` wich display hello microverse in your `index.html` file I think you should check well the requirement for this project and when ready you can come back, kindly fix this for your project to match requirements :wink:

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @lucy-sees  :wave:

 Great work on making the changes requested by a previous reviewer clap You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.
 
  

## Highlights 
 :heavy_check_mark: Good commit history.
 :heavy_check_mark: No linter errors. 
:heavy_check_mark: Your project is well documented 
:heavy_check_mark: Good PR description.

 ## Required Changes 
 _Check the comments under the review._
 
check below the review :point_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
 Cheers and Happy coding! :clap: :clap: :clap:

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lucy-sees** [comment]
Everything is now up to date as suggested by my last two code reviewers. Kindly approve my work for submission, I have made all the necessary changes.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 Hi @lucy-sees  :tada:  !
 
You have successfully completed this project! There is nothing else to say other than... it's time to merge it! rocket Time to celebrate, you have done a fantastic job, congratulations! :tada:

  
 ##  Highlights 
 * Great work using the right GitHub Flow :100:
 * Awesome file organization :card_file_box:
 * Commits are descriptive, good work.
 * Linters are working properly, great job!

 APPROVED! :100: :100: :100:

## Optional suggestions 
 _Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 100_
 
 * [ ]  **[OPTIONAL]:
 -N/A

Cheers and best of luck with the following projects! :raised_hands:
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me **@alfredbis29** in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — COMMENTED** [review]
Hi @lucy-sees   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Project has already been approved ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**